solutions.js

```diff
@@ -0,0 +1,46 @@
+//1
+class Human {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+  }
+}
+Human.prototype.ageOneYear = function () {
+  this.age++;
+};
+
+Human.prototype.eating = function () {
+  console.log("mmm, mmm, mmm, I'm love'n it");
+};
+
+let me = new Human('Mateo', 99);
+console.log(me.age);
+me.ageOneYear();
+console.log(me.age);
+me.eating();
+
+//2
+class Vector {
+  constructor(x, y) {
+    this.x = x;
+    this.y = y;
+  }
+}
+Vector.prototype.plus = function (vector) {
+  return new Vector(this.x + vector.x, this.y + vector.y);
+};
+
+Vector.prototype.minus = function (vector) {
+  return new Vector(this.x - vector.x, this.y - vector.y);
+};
+
+Vector.prototype.getLength = function () {
+  return Math.hypot(this.x, this.y);
+};
+
```

Comment:
Everything is great. Bonus points for clear argument names. Next time keep it consistent between ES5 and ES6, preferably ES6.